src/core.ts

```diff
@@ -135,7 +135,7 @@ function envelope(): IEnvelope {
   return {
     clarityId: cid,
     impressionId,
-    url: top.location.href,
+    url: location.href,
```

Comment:
Is window.location supported in all browsers? See:

![image](https://cloud.githubusercontent.com/assets/20712182/26754670/ca1f913e-484c-11e7-82e3-7ddbf12d3887.png)

From: https://developer.mozilla.org/en-US/docs/Web/API/Window/location